lighthouse-core/report/v2/report-renderer.js

```diff
@@ -251,9 +251,23 @@ class ReportRenderer {
   _renderCategory(category) {
     const element = this._createElement('div', 'lighthouse-category');
     element.appendChild(this._renderCategoryScore(category));
-    for (const audit of category.audits) {
+
+    const passedAudits = category.audits.filter(audit => audit.score === 100);
+    const nonPassedAudits = category.audits.filter(audit => !passedAudits.includes(audit));
+
+    for (const audit of nonPassedAudits) {
       element.appendChild(this._renderAudit(audit));
     }
+
+    const passedElem = this._createElement('details', 'lighthouse-category-passed');
+    const passedSummary = this._createElement('summary', 'lighthouse-cateogry-passed-summary');
+    passedSummary.textContent = `View ${passedAudits.length} passed items`;
+    passedElem.appendChild(passedSummary);
+
+    for (const audit of passedAudits) {
+      passedElem.appendChild(this._renderAudit(audit));
+    }
+    element.appendChild(passedElem);
```

Comment:
could also combine with the creation above:
`const passedElem = element.appendChild(this._createElement('details', 'lighthouse-category-passed'));`